hummingbot/connector/exchange/gate_io/gate_io_user_stream_tracker.py

```diff
@@ -27,13 +24,14 @@ def __init__(self,
                  gate_io_auth: Optional[GateIoAuth] = None,
                  trading_pairs: Optional[List[str]] = None,
                  api_factory: Optional[WebAssistantsFactory] = None):
-        super().__init__()
         self._api_factory = api_factory
         self._gate_io_auth: GateIoAuth = gate_io_auth
         self._trading_pairs: List[str] = trading_pairs or []
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
+        super().__init__(data_source=GateIoAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by `super().__init__`. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.